modules/common/src/main/base/PimpedUtils.scala

```diff
@@ -67,6 +67,12 @@ final class PimpedDateTime(private val date: DateTime) extends AnyVal {
   def toNow = new Duration(date, DateTime.now)
 }
 
+final class PimpedInstant(private val instant: Instant) extends AnyVal {
```

Comment:
Looks like these additions aren't used. (And `getSeconds` and `getCentis` is anyway a bit dubious for points in time, even on `DateTime`).